.ci/pipeline/ci.yml

```diff
@@ -15,20 +15,12 @@
 #===============================================================================
 
 variables:
-  - name: WINDOWS_BASEKIT_URL
```

Comment:
Why to remove this?

Reply:
It was not mentioned anywhere in code.

Reply:
hmm - interesting. ok I thought it was used - bight be in oneDAL ci